extension/services/spotify/player.js

```diff
@@ -30,8 +30,6 @@ this.player = (function() {
         try {
           const playerButton = await this.waitForSelector(SEARCH_PLAY, {
             timeout: 10000,
-            // There is no need to wait for this as there is only one selector
-            minCount: 0,
           });
```

Comment:
What was the initial idea behind adding minCount?

Reply:
It used to be `minCount: 4`, which meant only return when there were at least 4 elements that matched the selector. (For reasons I never looked into closely, Spotify had several buttons that matched the previous selector but didn't all work.)

Reply:
Ohh. I see